positron/modules/process.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Cc = Components.classes;
+const Ci = Components.interfaces;
+const Cr = Components.results;
+const Cu = Components.utils;
+
+Cu.import('resource://gre/modules/Services.jsm');
+
+// In Node, process.js is basically a noop, as it just re-exports the existing
+// `process` global.  But in Positron, we implement the `process` global via
+// this module, so we construct the `process` global by importing this module
+// rather than the other way around.
+
+// Because we construct the `process` global in this module, and we also inject
+// the global into this module, we don't actually have to export its symbols
+// here.  We just have to attach them to this module's own `process` global.
+// XXX Figure out if that's really the best way to implement this functionality.
+
+// This comes from Electron, where it's defined by common/init.js, which we
+// currently don't load.  Once we enable loading of init.js modules, we should
+// remove it from this module.
+process.atomBinding = function(name) {
+  try {
+    return process.binding("atom_" + process.type + "_" + name);
+  } catch (error) {
+    if (/No such module/.test(error.message)) {
+      return process.binding("atom_common_" + name);
+    } else {
```

Comment:
Indeed. This file was moved to this location in #2, so that's where I've fixed this in 37d59f9.